### DISCUSSION TITLE
本来想加的一个整点时刻弹一个通知的功能

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#108] [杂谈]
一直想要一个整点时刻手机发条通知的功能，主要是想培养对时间段的敏感性。 
手机经常熄屏在身边，需要在某些整点时刻，比如十点整，十一点整，收到一条通知点亮屏幕。  
找了一些代码和应用都试了下，有些时间不准，有些体量太大。  
最后使用系统自带的日历实现了，还挺自由的，也非常准时，好评！
  
![image](https://github.com/user-attachments/assets/37f324e2-437d-4079-bebc-15922d9e47d1)

**Comment by mimilinux** [comment]
记本怎么突然编辑不了背景和封面了啊，版本也已经是最新的了，救救孩子

**Reply by WatsonYao** [reply]
录屏发邮箱看看步骤

**Reply by mimilinux** [reply]
发了୧꒰•̀ᴗ•́꒱୨

**Reply by WatsonYao** [reply]
权限问题吧，悬浮窗、后台启动什么的，系统权限里面查查看
